### ISSUE TITLE
rework state and data flow with new endpoints

### ISSUE THREAD

**Issue by mirshko** [Uniswap/uniswap-info#21]
- [x] refactor data flow into state using unstated
- [x] clean up ui elements to match closer to the design
- [x] improve loading of transactions and charts with new endpoints

for a new ticket
- [ ] add in Uniswap Info homepage aggregating data
- [ ] implement routing

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 150.0 DAI (150.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $53,586.16 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 4 weeks, 1 day from now.
Please review their action plans below:


**1) [iamonuwa](https://gitcoin.co/profile/iamonuwa) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?mutate_worker_action=approve&worker=iamonuwa) | [reject worker](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?mutate_worker_action=reject&worker=iamonuwa))_.**

Requirements for this task are quite clear. I would love to work on getting this done.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315).

**Comment by eswarasai** [comment]
@conlan @haydenadams -- Can I get access to the designs in order to achieve the task `* clean up ui elements to match closer to the design`? Thanks!

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by mirshko** [comment]
still working!

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by mirshko** [comment]
still working!

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by mirshko** [comment]
still working!

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by mirshko** [comment]
still working!

**Comment by gitcoinbot** [comment]
@mirshko Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=1) | [3 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=3) | [5 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=5) | [10 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=10) | [100 days](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315?snooze=100)

**Comment by mirshko** [comment]
still working!

**Comment by ceresstation** [comment]
Hey @mirshko still good?

**Comment by mirshko** [comment]
yup! basically waiting on @conlan to finish up some api stuff and it’s good to launch. some bad values are keeping this delayed sadly.

**Comment by mirshko** [comment]
work should be completed in #26 #22 @ceresstation we're gonna move the other work to a new issue so we should prob close this out and then we can make a new one!

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 150.0 DAI (150.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@mirshko](https://gitcoin.co/profile/mirshko)



@ceresstation please take a look at the submitted work:
* [PR](https://github.com/Uniswap/uniswap-info/pull/22) by @mirshko

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $49,657.76 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 150.0 DAI (150.0 USD @ $1.0/DAI)  attached to this issue has been approved & issued to @mirshko.__  

 
* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Uniswap/uniswap-info/21/2315)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $49,217.02 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
